commands/operator-sdk/cmd/build.go

```diff
@@ -56,18 +137,78 @@ func buildFunc(cmd *cobra.Command, args []string) {
 	}
 
 	bcmd := exec.Command(build)
+	bcmd.Env = append(os.Environ(), fmt.Sprintf("TEST_LOCATION=%v", testLocationBuild))
+	bcmd.Env = append(bcmd.Env, fmt.Sprintf("ENABLE_TESTS=%v", enableTests))
 	o, err := bcmd.CombinedOutput()
 	if err != nil {
 		cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("failed to build: (%v)", string(o)))
 	}
 	fmt.Fprintln(os.Stdout, string(o))
 
+	namespacedRolesBytes := make([]byte, 0)
+	genWarning := ""
 	image := args[0]
-	dbcmd := exec.Command(dockerBuild)
-	dbcmd.Env = append(os.Environ(), fmt.Sprintf("IMAGE=%v", image))
+	intermediateImageName := image
+	if enableTests {
+		if namespacedManBuild == "" {
```

Comment:
I wonder if pulling this code block into its own function may make this more readable?